builtin/settingtypes.txt

```diff
@@ -952,6 +952,13 @@ serverlist_url (Serverlist URL) string servers.minetest.net
 #    Use this to stop players from being able to use color in their messages
 strip_color_codes (Strip color codes) bool false
 
+[*Stacksizes]
+
+#    Specifies the default stack size for nodes
+stacksize_default_node (Default node stack size) int 99
```

Comment:
this split doesn't really make sense necessarily, and shouldn't be exposed to users. Also, note that nodes are items too.

`default_stack_max` would be a better name imo